app/src/main/java/io/gnosis/safe/ui/transactions/execution/TxReviewFragment.kt

```diff
@@ -203,34 +213,50 @@ class TxReviewFragment : BaseViewBindingFragment<FragmentTxReviewBinding>() {
 
                             if (decodedData.method.lowercase() == "multisend") {
```

Comment:
this whole method looks complicated and difficult to understand w/o any comments & many nested blocks. Maybe extracting each case into a separate method would make the complexity lower?

Reply:
I have created a task for this [#1962](https://github.com/safe-global/safe-android/issues/1962)